.github/workflows/remove-stale-branches.yaml

```diff
@@ -2,7 +2,10 @@ name: "[internal] Remove Stale Branches"
 
 on:
   schedule:
-    - cron: "0 0 * * *" # Runs every night at midnight
```

Comment:
think you can just set this as
0 */6 * * *
which is the equivalent to every 6 hours

Reply:
Thank you, updated